superset/sql_parse.py

```diff
@@ -278,7 +285,11 @@ def _extract_from_token(  # pylint: disable=too-many-branches
         table_name_preceding_token = False
 
         for item in token.tokens:
-            if item.is_group and not self._is_identifier(item):
+            if (item.is_group and not self._is_identifier(item)) or (
```

Comment:
`item.is_group and (not self.is_identifier(item) or isinstance(item.tokens[0], Parenthesis))`?

Reply:
i think it's equivalent, do you think it's as readable?